tests/gold_tests/tls/ssl-post.c

```diff
@@ -134,15 +134,15 @@ spawn_same_session_send(void *arg)
     ret = select(sfd + 1, &reads, &writes, NULL, NULL);
     if (FD_ISSET(sfd, &reads) || FD_ISSET(sfd, &writes)) {
       ret = write_ret = SSL_write(ssl, req_buf, strlen(req_buf));
-      if (write_ret >= 0)
+      if (write_ret > 0)
         post_write_ret = SSL_write(ssl, post_buf, sizeof(post_buf));
     }
   }
 
-  while (write_ret < 0) {
```

Comment:
Seems like this could go into an infinite loop if the connection is closed. https://www.openssl.org/docs/man1.1.1/man3/SSL_write.html

Reply:
I agree, however, wasn't that the case prior to the change? I'm assuming that since this is a test at some point the CI will timeout and fail the tests. If you'd like to see the proper error handling added here, this will be a more significant change. I would also wonder, would you prefer I remove the changes to this test instead?